src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_commons.py

```diff
@@ -214,34 +243,26 @@ def read_ndarray_from_tsv(input_file: str,
     """
     dtype = None
     shape = None
-    rows: List[np.ndarray] = []
-
-    def _get_value(key: str, _line: str):
-        key_loc = _line.find(key)
-        if key_loc >= 0:
-            val_loc = _line.find('=')
-            return _line[val_loc + 1:].strip()
-        else:
-            return None
 
     with open(input_file, 'r') as f:
         for line in f:
             stripped_line = line.strip()
             if len(stripped_line) == 0:
                 continue
             elif stripped_line[0] == comment:
-                if dtype is None:
-                    dtype = _get_value('dtype', stripped_line)
-                if shape is None:
-                    shape = _get_value('shape', stripped_line)
+                key, value = parse_sam_comment(stripped_line)
```

Comment:
Might be worth checking that there aren't multiple dtype/shape lines present?

Reply:
I agree, done